package.json

```diff
@@ -2,47 +2,46 @@
   "dependencies": {
     "@react-google-maps/api": "^2.18.1",
     "clsx": "^1.2.1",
-    "next": "13.2.1",
-    "pnp": "^0.0.3",
-    "pnpm": "^7.27.1",
```

Comment:
I don't believe we need `pnpm` under `dependencies`, and I wonder if this was giving anybody grief?